cms/templates/js/access-editor.underscore

```diff
@@ -0,0 +1,45 @@
+<form>
+    <% if (xblockInfo.get('prereqs').length > 0) { %>
+        <h3 class="modal-section-title"><%= gettext('Limit Access') %></h3>
+        <div class="modal-section-content gating-prereq">
+            <ul class="list-fields list-input">
+                <p class="field-message">
+                    <%- gettext('Select a prerequisite subsection and enter a minimum score percentage to limit access to this subsection.') %>
+                </p>
+                <li class="field field-select">
+                    <label for="prereq" class="label">
+                        <%- gettext('Prerequisite:') %>
+                    </label>
+                    <select id="prereq" class="input">
+                        <option value="">No prerequisite</option>
+                        <% _.each(xblockInfo.get('prereqs'), function(prereq){ %>
+                            <option value="<%- prereq.block_usage_key %>"><%- prereq.block_display_name %></option>
+                        <% }); %>
+                    </select>
+                </li>
+                <li id="prereq_min_score_input" class="field field-input input-cosmetic">
+                    <label for="prereq_min_score" class="label">
```

Comment:
You can nest the input within the label to avoid the need for linking by ID. This is covered in the accessibility edX course.

For example,
````
<label>
   <input type="text" name="lastname" />
   Last Name
</label>
````